src/apps/Organization/Organization.js

```diff
@@ -335,11 +334,7 @@ const Organization = React.memo(function Organization({
             </Info>
           ) : (
             <Info mode="warning">
-              {`Please ${
-                wallet.networkType !== network.type
-                  ? `select the ${sanitizeNetworkType(network.type)} network`
-                  : 'unlock your account'
-              } in ${getProviderString(
+              {`Please unlock your account in ${getProviderString(
```

Comment:
Similar to the Account module, I've simplified places where we check the connect account's network vs. the expected network as the user currently can never have their account enabled on a different network.